### PR TITLE
Python - Add codebytes for math.pow and math.sin

### PR DESCRIPTION
**Pull request by AapoSoukkio**
### Description

- Added Codebytes to the listed entries.
   https://www.codecademy.com/resources/docs/python/math-module/math-pow
   https://www.codecademy.com/resources/docs/python/math-module/math-sin
- Closes Issue  [#2886](https://github.com/Codecademy/docs/issues/2886)

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-sin/math-sin.md [hunk 1]
```diff
@@ -43,3 +43,17 @@ print(math.sin(5.5))
 
 # Output: -0.7055403255703919
 ```
+
+## Codebyte Example
+
+Return the sine of different numbers
```
Comment (caupolicandiaz) [inline content/python/concepts/math-module/terms/math-sin/math-sin.md:49]:
```suggestion
The following code is runnable and returns the sine of different values:
```

File: content/python/concepts/math-module/terms/math-pow/math-pow.md [hunk 2]
```diff
@@ -45,3 +45,13 @@ print(math.pow(5.5, 3.3))
 
 # Output: 277.457759723262
 ```
+
+## Codebyte Example
+
+Return the value of 4 raised to the power of 3
```
Comment (caupolicandiaz) [inline content/python/concepts/math-module/terms/math-pow/math-pow.md:51]:
```suggestion
The following code is runnable and demonstrates the use of `math.pow()`:
```

File: content/python/concepts/math-module/terms/math-pow/math-pow.md [hunk 3]
```diff
@@ -45,3 +45,13 @@ print(math.pow(5.5, 3.3))
...
+
+```codebyte/py
+import math
+
+print(math.pow(4, 3))
```
Comment (caupolicandiaz) [inline content/python/concepts/math-module/terms/math-pow/math-pow.md:56]:
```suggestion
print(math.pow(2, 0.0))
print(math.pow(1.0, 2))
```

Comment (caupolicandiaz) [reply content/python/concepts/math-module/terms/math-pow/math-pow.md:56]: I updated this example to illustrate one of the special cases in order to differentiate it more from the other examples

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@AapoSoukkio I've completed my initial review. Please update the entries, and tag me when you've finished making changes. Thanks

**Comment by AapoSoukkio** [comment]
@caupolicandiaz Thank you for your review. The entries has been updated as requested now.

**Review by caupolicandiaz — APPROVED** [review]
@AapoSoukkio Looks good. I'm passing this on for a second review.

**Review by SSwiniarski — APPROVED** [review]
@AapoSoukkio Looks good.

**Comment by SSwiniarski** [comment]
@AapoSoukkio Congrats on the new edits!

Live links:
- https://www.codecademy.com/resources/docs/python/math-module/math-pow
- https://www.codecademy.com/resources/docs/python/math-module/math-sin

**Pull request merged**